src/org/pahappa/systems/ticketing/models/Ticket.java

```diff
@@ -1,11 +1,34 @@
 package org.pahappa.systems.ticketing.models;
 
+import org.pahappa.systems.ticketing.constants.TicketStatus;
+
 /**
  * A ticket refers to a unit of work or a request that is submitted
- * by a user or customer to seek assistance, report an issue, or request a service.
- * It serves as a record or a container that captures all the relevant information
+ * by a user or customer to seek assistance, report an issue, or request a
+ * service.
+ * It serves as a record or a container that captures all the relevant
+ * information
  * related to the user's request or issue.
  */
 public class Ticket {
+    public String client;
+    public String contact;
+    public String category;
+    public String description;
+    public TicketStatus status;
+    public String priorityLevel;
```

Comment:
Apply encapsulation to this class.

https://beginnersbook.com/2013/05/encapsulation-in-java/